public/js/monitor.js

```diff
@@ -95,25 +124,39 @@ async function sha1(message) {
 
 async function hashEmailAndSend(emailFormSubmitEvent) {
   emailFormSubmitEvent.preventDefault();
-  handleEvents("Scan");
   const emailForm = emailFormSubmitEvent.target;
+  if(emailForm.querySelector("input[name=email]").value === ""){
+    emailForm.querySelector("input[type=submit").disabled = true;
```

Comment:
something looks 🐟 here.
I think you're missing a closing `]` after the `[type=submit` here.

Reply:
shameful!

Reply:
![](http://www.reactiongifs.com/r/shm.gif)

Reply:
hahahaha basically.